deploy/osd-suricata/hypershift-management-cluster/20-osd-suricata-Daemonset.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: suricata
+  namespace: openshift-suricata
+spec:
+  selector:
+    matchLabels:
+      name: suricata
+  template:
+    metadata:
+      labels:
+        name: suricata
+    spec:
+      containers:
+        - env:
+            - name: OO_PAUSE_ON_START
+              value: "false"
+          image: quay.io/app-sre/suricata@sha256:2e7ff9481e89b521d5d17ec9d17752d65684dcb616bbaafb442dae3eb5b2ed66
+          imagePullPolicy: IfNotPresent
+          name: suricata
+          resources:
+            limits:
+              cpu: 900m
+              memory: 900Mi
+            requests:
+              cpu: 100m
+              memory: 50Mi
+          securityContext:
+            capabilities:
+              add: ["NET_ADMIN", "NET_RAW"]
+          volumeMounts:
+            - mountPath: /host/
+              name: host-filesystem
+            - mountPath: /etc/suricata/rules/sre-events.rules
+              subPath: "sre-events.rules"
+              name: suricata-config
+            - mountPath: /etc/suricata/suricata.yaml
+              subPath: "suricata.yaml"
+              name: suricata-config
+        - name: log-cleaner
+          image: quay.io/app-sre/log-cleaner@sha256:5796aba1ff3f592cadda48de61084d7d570b6ed4dc28185514a5856afa8c49c5
+          volumeMounts:
+            - mountPath: /host/
+              name: host-filesystem
+      dnsPolicy: ClusterFirst
+      hostNetwork: true
+      nodeSelector:
+        kubernetes.io/os: linux
+      serviceAccountName: suricata-sa
+      tolerations:
+        - operator: Exists
```

Comment:
Can you please add tolerations to avoid the following nodes?
```
node-role.kubernetes.io/master
node-role.kubernetes.io/infra
```
This way, this can be deployed to default worker pools and future request serving nodes.